### PR TITLE
Various tweaks

### PR DESCRIPTION
**Pull request by nautatva**
Electron 8.0.0 is out. https://github.com/sindresorhus/caprine/issues/14.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#14: Implement spell checking](https://issuehunt.io/repos/42574339/issues/14)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

File: source/index.ts [hunk 1]
```diff
@@ -285,6 +286,7 @@ function createMainWindow(): BrowserWindow {
 		webPreferences: {
 			preload: path.join(__dirname, 'browser.js'),
 			contextIsolation: true,
+			spellcheck: true,
```
Comment (sindresorhus) [inline source/index.ts]: I'm not sure about this. When the user sees the red squiggly lines, they assume they can right-click to get suggestion for spelling, which currently won't happen. I think we should wait for https://github.com/sindresorhus/electron-context-menu/issues/11#issuecomment-582300971.

**Comment by sindresorhus** [comment]
Sorry about the merge conflict. I was already working on updating the dependencies.

File: source/index.ts (hunk 1 above)
Comment (nautatva) [reply source/index.ts]: Trying to work on it. Will submit a pull request when i figure it out

Comment (nautatva) [reply source/index.ts]: Check out the PR https://github.com/sindresorhus/electron-context-menu/pull/94. This was my first time working with any library. Took some time to understand the code, but enjoyed working on it🎉

**Comment by nautatva** [comment]
- [x] Resolve the merge conflict.

**Pull request merged**